install/instradaogm-install.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: rdeangel
+# License: MIT | https://github.com/community-scripts/ProxmoxVED/raw/main/LICENSE
+# Source: https://github.com/rdeangel/InstradaOGM
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+NODE_VERSION="23" NODE_MODULE="pm2" setup_nodejs
+fetch_and_deploy_gh_release "instradaogm" "rdeangel/InstradaOGM" "tarball"
+
+ensure_dependencies python3 sqlite3 ca-certificates jq
```

Comment:
```suggestion
ensure_dependencies python3 sqlite3 ca-certificates
```
@MickLesk is that correct use of the func like this?

jq is anyway core?